airflow/www/templates/airflow/dag_code.html

```diff
@@ -22,34 +22,33 @@
 {% block page_title %}{{ dag.dag_id }} - Code - Airflow{% endblock %}
 
 {% block content %}
-    {{ super() }}
-    <h4>{{ title }}</h4>
-    <div class="active">
-      <a onclick="toggleWrap()">Toggle wrap</a>
-    </div>
+  {{ super() }}
+  <div class="code-wrap">
+    <a onclick="toggleWrap()" class="btn btn-default code-wrap-toggle">Toggle Wrap</a>
     {{ html_code }}
+  </div>
 {% endblock %}
 
-{% block tail %}
-    {{ super() }}
-    <script>
-      function toggleWrap() {
-        $('.code pre').toggleClass('wrap')
-      };
-
-      // We blur task_ids in demo mode
-      $( document ).ready(function() {
-        if ("{{ demo_mode }}" == "True") {
-            $("pre span.s").css({
-                'text-shadow': '0px 0px 10px red',
-                'color': 'transparent',
-            });
-        }
-      });
-
-      // pygments generates the HTML so set wrap toggle via js
-      if ("{{ wrapped }}" == "True") {
-        toggleWrap();
-      };
-    </script>
+{% block tail_js %}
```

Comment:
Any reason for different block in use here?

Reply:
Just attempting to make the templates more consistent—not sure if I got them all. Presumably the `tail_js` comes after the `tail` as it's best practice for the scripts to be at the end of the document. Also assuming that the "js" in the block name designates the purpose.

Reply:
Gotcha :+1: